buildpack.toml

```diff
@@ -16,5 +16,8 @@ api = "0.7"
 [[stacks]]
   id = "io.buildpacks.stacks.bionic"
 
+[[stacks]]
+  id = "io.buildpacks.stacks.jammy"
+
```

Comment:
Maybe use a wildcard stack id?

Reply:
I think a wildcard should work. After all, what it really means is that `pipenv-install` should work wherever its dependencies are met.

Reply:
done